src/Compute/Compute/ChangeLog.md

```diff
@@ -19,7 +19,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Added `vmHealh` property to ``Get-AzVm ... -Status``'s return object (VMInstanceView)
+* Added ``vmHealh`` property to the virtual machine's instance view, which is the returned object when ``Get-AzVm`` is invoked with ``-Status``
```

Comment:
Please correct the spelling to VmHealth as I commented before.

Reply:
Just checked. The field name is VMHealth so I am changing to that.